### PR TITLE
pipeline: filters: sysinfo: add document for sysinfo plugin

### PR DESCRIPTION
**Pull request by nokute78**
This PR is to add a document for filter_sysinfo plugin.
https://github.com/fluent/fluent-bit/pull/8106

### CODE REVIEW

File: pipeline/filters/sysinfo.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+|hostname_key|Specify the key name for hostname.| All|
+|os_version_key|Specify the key name for os version. It is not supported on some platforms. | Linux |
+|kernel_version_key|Specify the key name for kernel version. It is not supported on some platforms.| Linux |
+
+
+## Getting Started
+
+In order to start filtering records, you can run the filter from the command line or through the configuration file.
+
+
```
Comment (patrick-stephens) [inline pipeline/filters/sysinfo.md:24]:
```suggestion
```

File: pipeline/filters/sysinfo.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Sysinfo
+
+The _Sysinfo Filter_ plugin allows to append system information.
```
Comment (patrick-stephens) [inline pipeline/filters/sysinfo.md:3]: Can we expand a little more on what that is?

File: pipeline/filters/sysinfo.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Configuration Prameters
+
+The plugin supports the following configuration parameters:
+
+Some properties are supported by specific platform.
```
Comment (patrick-stephens) [inline pipeline/filters/sysinfo.md:9]: Let's move below the table

**Comment by nokute78** [comment]
Updated.

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**